here_location_services/ls.py

```diff
@@ -233,6 +242,81 @@ def calculate_isoline(
             raise ValueError("Isolines could not be calculated.")
         return response
 
+    def autosuggest(
+        self,
+        q: str,
+        at: Optional[List] = None,
+        search_in_circle: Optional[SearchCircle] = None,
+        search_in_box: Optional[SearchBox] = None,
+        in_country: Optional[List] = None,
+        limit: Optional[int] = None,
+        terms_limit: Optional[int] = None,
+        lang: Optional[List] = None,
+        political_view: Optional[str] = None,
+        show: Optional[List] = None,
+    ) -> AutosuggestResponse:
+        """Suggest address or place candidates based on an incomplete or misspelled query
+
+        :param q: A string for free-text query. Example: res, rest
+        :param at: Specify the center of the search context expressed as coordinates
```

Comment:
`expressed as list of coordinates`

Reply:
so as far as I understood from the documents, there can only be one center for a particular search.
https://developer.here.com/documentation/geocoding-search-api/api-reference-swagger.html